<ui version="4.0" >
 <class>CaloClusterSysSettingsCutsForm</class>
 <widget class="QWidget" name="CaloClusterSysSettingsCutsForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Energy</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="leftMargin" >
       <number>4</number>
      </property>
      <property name="topMargin" >
       <number>4</number>
      </property>
      <property name="rightMargin" >
       <number>4</number>
      </property>
      <property name="bottomMargin" >
       <number>4</number>
      </property>
      <item>
       <layout class="QGridLayout" >
        <property name="verticalSpacing" >
         <number>2</number>
        </property>
        <item row="0" column="0" >
         <widget class="QCheckBox" name="checkBox_Emin" >
          <property name="toolTip" >
           <string>Check this box to enable a lower cut on cluster energies</string>
          </property>
          <property name="text" >
           <string>Min energy:</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QDoubleSpinBox" name="doubleSpinBox_Emin" >
          <property name="toolTip" >
           <string>Value of lower cut on cluster energies</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix" >
           <string> GeV</string>
          </property>
          <property name="maximum" >
           <double>999.990000000000009</double>
          </property>
          <property name="singleStep" >
           <double>0.100000000000000</double>
          </property>
          <property name="value" >
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QCheckBox" name="checkBox_Emax" >
          <property name="toolTip" >
           <string>Check this box to enable an upper cut on cluster energies</string>
          </property>
          <property name="text" >
           <string>Max energy:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QDoubleSpinBox" name="doubleSpinBox_Emax" >
          <property name="toolTip" >
           <string>Value of upper cut on cluster energies</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix" >
           <string> GeV</string>
          </property>
          <property name="maximum" >
           <double>999.990000000000009</double>
          </property>
          <property name="singleStep" >
           <double>0.100000000000000</double>
          </property>
          <property name="value" >
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Direction</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="leftMargin" >
       <number>4</number>
      </property>
      <property name="topMargin" >
       <number>4</number>
      </property>
      <property name="rightMargin" >
       <number>4</number>
      </property>
      <property name="bottomMargin" >
       <number>4</number>
      </property>
      <item>
       <widget class="VP1EtaPhiCutWidget" native="1" name="etaPhiCutWidget" />
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Presets:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_quicketa_barrel" >
          <property name="toolTip" >
           <string>Click to change eta cuts to include just the barrel region</string>
          </property>
          <property name="text" >
           <string>Brl</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_quicketa_endcapA" >
          <property name="toolTip" >
           <string>Click to change eta cuts to include just the A-side endcap</string>
          </property>
          <property name="text" >
           <string>EcA</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_quicketa_endcapC" >
          <property name="toolTip" >
           <string>Click to change eta cuts to include just the C-side endcap</string>
          </property>
          <property name="text" >
           <string>EcC</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>1</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VP1EtaPhiCutWidget</class>
   <extends>QWidget</extends>
   <header>VP1Base/VP1EtaPhiCutWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
